05_stringer/runnerdave/main_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("127.0.0.1\n")
+	actual := strconv.Quote(buf.String())
+	t.Logf("expected:%s", expected)
+	t.Logf("actual:%s", actual)
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()")
+	}
+}
+
+func TestFibo(t *testing.T) {
```

Comment:
Is it really testing fibonacci? ;)

Totally a nitpick, but good to watch out for such things when copy-pasting especially if they're publicly exported.

(Or totally avoid copy-pasting when first acclimatising to a new language)

Reply:
Please change this.

Reply:
hehe good catch